lib/transactions/crypto.js

```diff
@@ -1,24 +1,44 @@
-var crypto = require("crypto-browserify");
-var constants = require("../constants.js");
+/**
+ * Crypto module helping with cryptograhpy in Lisk Blockchain
```

Comment:
Crypto module provides functions for cryptographic functions in the Lisk Blockchain